packages/core/src/test/testRunner.ts

```diff
@@ -31,20 +31,40 @@ export async function runTests(
 
     function getRoot(): string {
         const abs = process.env['DEVELOPMENT_PATH'] ?? process.cwd()
-
         if (process.platform !== 'win32') {
             return abs
         }
 
-        // Force all drive letters (or whatever else is before the first colon) to be lowercase.
-        //
-        // Node's `require` will always cache modules based off case-sensitive paths, regardless
-        // of the underlying file system. This is normally not a problem, but VS Code also happens
-        // to normalize paths on Windows to use lowercase drive letters when using its bootstrap loader.
-        // This means that each module ends up getting loaded twice, once by the extension and once
-        // by any test code, causing all sorts of bizarre behavior during tests.
+        /**
+         * Node's `require` will always cache modules based off case-sensitive paths, regardless
```

Comment:
you mentioned upstream vscode issues, are there any good ones we can reference here?

Reply:
I couldn't find any that were relevant to this specific comment. I'm not sure if Jaden looked through the code to figure that out or not. The ones I found just referenced issues with breakpoints and debug paths and a couple other random internal things